packages/diagram/package.json

```diff
@@ -45,6 +45,7 @@
     "clean": "run -T rimraf -v -g 'dist/*' .tsbuildinfo"
   },
   "dependencies": {
+    "@ant-design/colors": "^7.1.0",
```

Comment:
Should be moved to `core` where it is used actually

Reply:
Done